internal/runners/projects/edit.go

```diff
@@ -87,7 +91,7 @@ func (e *Edit) Run(params EditParams) error {
 	}
 	editMsg += locale.Tl("edit_prompt_confirm", "Continue?")
 
-	edit, err := e.prompt.Confirm("", editMsg, p.BoolP(false))
+	edit, err := e.prompt.Confirm("", editMsg, &e.out.Config().Interactive)
```

Comment:
I think you meant to use a `!` prefix here. The default should be false for interactive mode, and true for non-interactive mode. Also, is it safe to put the pointer to the actual mode here? Normally I would do something like `default := !e.out.Config().Interactive` and then `&default` in order to avoid possibly changing the underlying config value.